lib/datadog/tracing/configuration/ext.rb

```diff
@@ -54,7 +54,8 @@ module Transport
 
         # @public_api
         module ClientIp
-          ENV_DISABLED = 'DD_TRACE_CLIENT_IP_HEADER_DISABLED'.freeze
+          ENV_ENABLED = 'DD_TRACE_CLIENT_IP_HEADER_ENABLED'.freeze
+          ENV_DISABLED = 'DD_TRACE_CLIENT_IP_HEADER_DISABLED'.freeze # TODO: deprecated, remove later
```

Comment:
I think there might be a mistake in the original one and it's supposed to be `DD_TRACE_CLIENT_IP_ENABLED`. Checking.